admission-webhook/go.mod

```diff
@@ -1,32 +1,34 @@
 module github.com/kubernetes-sigs/windows-gmsa/admission-webhook
 
-go 1.21
+go 1.22.0
```

Comment:
should this be 1.22?

Reply:
When I run `go mod tidy` it is updated from 1.22 to 1.22.0. Must be new in golang 1.22 tooling